src/cosmoz-bottom-bar-focused-context.js

```diff
@@ -0,0 +1,26 @@
+import { createContext, useContext, useState, useMemo } from '@pionjs/pion';
+import { noop } from '@neovici/cosmoz-utils/function';
+
+export const bottomBarFocused = createContext({
+	bars: [],
+	add: noop,
+	remove: noop,
+});
+customElements.define('bottom-bar-focused-provider', bottomBarFocused.Provider);
```

Comment:
We do however need to prefix the element name, as that is registered globally.

```suggestion
customElements.define('cosmoz-bottom-bar-focus-provider', Focus.Provider);
```